tsconfig.json

```diff
@@ -9,6 +9,9 @@
     "noEmit": true,
     "noErrorTruncation": true,
     "noUncheckedIndexedAccess": true,
+    "paths": {
+      "@metamask/utils/node": ["./node_modules/@metamask/utils/dist/types/node"]
```

Comment:
The reason we're setting `paths` is explained here: https://github.com/MetaMask/module-lint/pull/20#discussion_r1373906853